doc/api_keys.md

```diff
@@ -38,6 +38,9 @@ be put into the app in the form of a resource file called `apikeys.xml` under
    <string name="nobil_key" translatable="false">
       insert your nobil key here
    </string>
+   <string name="evmap_key" translatable="false">
```

Comment:
I named this "evmap_key". Would you prefer to include nobil in the name? If you, for instance, want to have several APIs in your backend, each requiring a separate key.